### PR TITLE
Added Sign Up page

### PR DESCRIPTION
**Pull request by RomanMykush**
Please merge this to main.

### CODE REVIEW

File: src/assets/SignUpPage/styles/general.css
Comment (AlberdaNazarius) [inline src/assets/SignUpPage/styles/general.css:1]: Why have you added background-color to general. It's only applicable for your page not for every other

File: src/assets/SignUpPage/styles/sign_up_page.css
Comment (AlberdaNazarius) [inline src/assets/SignUpPage/styles/sign_up_page.css:1]: Position absolute is bad

**Review by AlberdaNazarius — CHANGES_REQUESTED** [review]

File: src/assets/SignUpPage/styles/sign_up_page.css
Comment (RomanMykush) [reply src/assets/SignUpPage/styles/sign_up_page.css:1]: This will do for now

File: src/assets/SignUpPage/styles/general.css
Comment (RomanMykush) [reply src/assets/SignUpPage/styles/general.css:1]: Commit c116ee3cd59cd0703697d4bfac4946b36adda0e0 fixes this

**Review by AlberdaNazarius — APPROVED** [review]

**Pull request merged**